lib/src/core/verifier/mod.rs

```diff
@@ -0,0 +1,76 @@
+use crate::*;
+use anyhow::Result;
+use async_trait::async_trait;
+use serde_json::{json, Value};
+use std::path::PathBuf;
+
+pub mod policy;
+mod sample;
+mod sgx;
+mod tdx;
+
+#[derive(Debug)]
+pub struct Evidence {
+    pub quote: String,
+    pub ehd: String,
+    pub add: String,
+}
+
+#[derive(Debug)]
+pub struct AttestationResults {
+    pub tee: String,
+    pub attestation_type: String,
```

Comment:
Why do we need both a `tee` and `attestation_type` field? Seems like these would be the same value?